torch/csrc/onnx/init.cpp

```diff
@@ -229,9 +229,10 @@ void initONNXBindings(PyObject* module) {
 
   m.def(
       "_check_onnx_proto",
-      [](const std::string& proto_string, bool full_check) {
-        check_onnx_proto(proto_string, full_check);
-      },
+      ::torch::wrap_pybind_function(
+          [](const std::string& proto_string, bool full_check) {
+            check_onnx_proto(proto_string, full_check);
+          }),
```

Comment:
Do we need a lambda here? Can this be `::torch::wrap_pybind_function(check_onnx_proto)`?

Just wondering, since there may be pybind limitations I am not aware of

Reply:
Tested, and it made no difference, so I modified it as suggested.

Reply:
Sorry, this is needed, since it passes parameter names to pybind and let users consume them in python. I think we should keep it.

Reply:
Can you share a link if there's a doc?

Reply:
> Do we need a lambda here? Can this be `::torch::wrap_pybind_function(check_onnx_proto)`?
> 
> Just wondering, since there may be pybind limitations I am not aware of

@justinchuby Do you know what `::torch::wrap_pybind_function(check_onnx_proto)`? Adding code we dont know what it does is dangerous. The safer bet is, if it is working and you dont know what the changes would cause, better not to touch it :)